plugins/sectionManager.js

```diff
@@ -0,0 +1,65 @@
+
+const fs = require('fs');
+const path = require('path');
+const marked = require('marked');
+const config = require('../getConfig').get();
+
+module.exports = {
+  
+  listSections: [],
+  
+  process(filename, data) {
+    let fileString = data.contents.toString();
+      
+    if (match = fileString.match(/(\[section=)[a-zA-Z0-9]+\]/g)) {
+      fileString = this.injectSection(match, filename, fileString);
+    }
+    
+    return {
+      has_section: (match) ? true : false,
+      sections: [...this.listSections].join(),
```

Comment:
Why do you duplicate the `listSections` array before joining it?
This should be simplified: `sections: this.listSections.join(),`